temporalcli/commandsgen/commands.yml

```diff
@@ -4093,6 +4144,9 @@ option-sets:
           HTTP headers for requests to codec server.
           Format as a `KEY=VALUE` pair.
           May be passed multiple times to set multiple headers.
+      - name: identity
+        type: string
+        description: The identity of the user or client submitting this request.
```

Comment:
May be worth clarifying that there is a default that includes `user@hostname`, but no biggie

Reply:
I think in an ideal world we could put the code here as a function but our generator isn't up to that. I'll remove the default field and put this in the description.